src/gesture.js

```diff
@@ -417,6 +426,14 @@ export class Gestures {
       }
     }
     if (cancelEvent) {
+      if (event.type == 'touchend') {
+        this.lastEventTarget_ = event.target;
```

Comment:
Maybe be more specific with this name, something like `this.lastTouchendTarget_`, since it is only assigned for `touchend` and not all events here